### DISCUSSION TITLE
功夫网抽风？将一堆网站都进行了TLS降级……

### DISCUSSION THREAD

**Discussion by daiaji** [v2fly/v2ray-core#519] [General]
```
curl -v https://www.cloudflare.com/
*   Trying 123.160.221.98:443...
* Connected to www.cloudflare.com (123.160.221.98) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: none
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS alert, handshake failure (552):
* error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure
* Closing connection 0
curl: (35) error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure


```
然后我去用TOR了……
TOR的网桥也被降级到了sslv3
测试了其他的网站，很多都降级了
这是想干嘛？

**Comment by rampageX** [comment]
你网络问题，我这电信移动都没问题，广州：

> curl -v -I https://www.cloudflare.com/
> *   Trying 104.16.124.96:443...
> * Connected to www.cloudflare.com (104.16.124.96) port 443 (#0)
> * successfully set certificate verify locations:
> *   CAfile: /etc/ssl/certs/ca-certificates.crt
>   CApath: none
> * ALPN, offering h2
> * ALPN, offering http/1.1
> * ALPN, server accepted to use h2
> * SSL connection using TLSv1.3 / TLS13-AES128-GCM-SHA256
> * Using HTTP2, server supports multi-use
> * Connection state changed (HTTP/2 confirmed)
> * Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
> * Using Stream ID: 1 (easy handle 0xa18d0)
> > HEAD / HTTP/2
> > Host: www.cloudflare.com
> > user-agent: curl/7.70.0
> > accept: */*
> >
> * Connection state changed (MAX_CONCURRENT_STREAMS == 256)!
> < HTTP/2 200
> HTTP/2 200

**Comment by rampageX** [comment]
> *   Trying 123.160.221.98:443... 

IP 就不对，`123.160.221.98:443` 归属 `CHINANET henan province network`,查下你的 DNS。

**Reply by daiaji** [reply]
> > * Trying 123.160.221.98:443...
> 
> IP 就不对，`123.160.221.98:443` 归属 `CHINANET henan province network`,查下你的 DNS。

只有一段时间，也许是dns投毒。
实际上等你测试的时候，网络已经恢复了。
